packages/@glimmer/syntax/lib/v2-a/objects/node.ts

```diff
@@ -60,7 +60,7 @@ export function node<T extends string>(
           constructor(fields: BaseNodeFields & Fields) {
             this.type = type;
             this.loc = fields.loc;
```

Comment:
looks like this line could be removed, I'm not clear why it's here before

Reply:
That's a good catch – I’ll test and see!

Reply:
oh, just bubbling is "loc" enumerable here? because if not, Object.assign won't work for it..

Reply:
![image](https://user-images.githubusercontent.com/1360552/134369751-c201e544-f708-4715-b7d7-b8af27d78038.png)
looks like we could define loc getter without calling it from fields and save some CPU

```js
Object.defineProperty(this, 'loc', { 
   get() {
    return fields.loc;
  }
});
```